model/map-hot-rod/src/main/java/org/keycloak/models/map/storage/hotRod/IckleQueryOperators.java

```diff
@@ -74,25 +76,31 @@
         /**
          * Produces an Ickle query where clause for obtained parameters
          *
+         * @param modelField searchable field
          * @param fieldName left side operand
          * @param values right side operands
          * @param parameters mapping between named parameters and actual parameter values
          * @return resulting string that will be part of resulting
          */
-        String combine(String fieldName, Object[] values, Map<String, Object> parameters);
+        String combine(SearchableModelField<?> modelField, String fieldName, Object[] values, Map<String, Object> parameters);
```

Comment:
We introduced this parameter to distinguish between analyzed and non-analyzed fields. However, currently, we are not using it, should we remove this parameter?

Reply:
yes, removed.